nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/connection/prioritizers/prioritizers.component.html

```diff
@@ -73,20 +84,34 @@
         </div>
     </div>
     <ng-template #prioritizerItem let-item let-i="i" let-canClose="canClose">
-        <div class="flex gap-x-3 items-center">
-            <div>{{ getPrioritizerLabel(item) }}</div>
-            @if (hasDescription(item)) {
-                <i
-                    class="fa fa-info-circle"
-                    nifiTooltip
-                    [tooltipComponentType]="TextTip"
-                    [tooltipInputData]="item.description"></i>
+        @if (!isDisabled) {
+            <div class="flex items-center">
+                <span class="grip pr-5"></span>
+                <div class="prioritizer-name" [title]="getPrioritizerLabel(item)">{{ getPrioritizerLabel(item) }}</div>
+                @if (hasDescription(item)) {
+                    <i
+                        class="pl-1 fa fa-info-circle surface-color"
+                        nifiTooltip
+                        [tooltipComponentType]="TextTip"
+                        [tooltipInputData]="item.description"></i>
+                }
+            </div>
+            @if (canClose) {
+                <button class="pr-1" type="button" (click)="removeSelected(item, i)" [disabled]="isDisabled">
```

Comment:
We no longer need to set `disabled` here since we only render the remove button when `!isDisabled`.